testcases/OpTestDlparIO.py

```diff
@@ -67,6 +70,20 @@ def setUp(self):
         self.loc_code = self.get_slot_from_sysfs(self.pci_device)
         self.get_slot_hw_details()
 
+    def adapter_check(self):
+        """
+        Check device type , configure ip and perform ping test
+        """
+        self.pci_device_type = self.util.get_pci_type(self.pci_device, self.cv_HOST)
+        log.info("Type of PCI device is \"%s\"", self.pci_device_type)
+        if self.pci_device_type == "network":
+            self_get_interface = self.util.get_interface(self.pci_device, self.cv_HOST)
+            log.info("PCI device mapped to interface: \"%s\"", self_get_interface)
+            if self.util.configure_host_ip(self_get_interface, self.test_ip, self.cv_HOST):
+                if self.util.ping_test(self_get_interface, self.peer_ip, self.cv_HOST):
```

Comment:
When you are checking the configure_host_ip and ping_test.. are these functions returning bolean ? you need to modify those functions too